### PR TITLE
Support AgentFile on service level

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

File: cloudify/agentfile.go [hunk 1]
```diff
@@ -52,8 +49,39 @@ func (cl *Client) updateHostFromAgent() {
...
+	}
+
+	if len(cloudConfig.User) == 0 {
+		return fmt.Errorf("You have empty user")
+	}
+
+	if len(cloudConfig.Password) == 0 {
+		return fmt.Errorf("You have empty password")
+	}
+
```
Comment (mabuaisha) [inline cloudify/agentfile.go:71]: @0lvin What about checking the agent/tenant values ?

**Review by mabuaisha — COMMENTED** [review]
@0lvin left one simple comment

File: cloudify/agentfile.go (hunk 1 above)
Comment (0lvin) [reply cloudify/agentfile.go:71]: We don't need to check "tenant" globally, you can use command like:
````
bin/cfy-go tenants list -host https://<some_host> -tenant "" -debug
````

Comment (mabuaisha) [reply cloudify/agentfile.go:71]: Got it :)

**Pull request merged**